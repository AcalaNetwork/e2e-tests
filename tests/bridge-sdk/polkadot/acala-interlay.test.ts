import { bridgeSDKTest } from '../shared'

bridgeSDKTest({
  from: 'acala',
  to: 'interlay',
  token: 'INTR',
})
