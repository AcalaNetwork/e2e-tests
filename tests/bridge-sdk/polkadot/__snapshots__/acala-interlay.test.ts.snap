// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`'acala' to 'interlay' using bridgeSDK cross-chain 'INTR' > after 1`] = `
{
  "address": "5FA9nQDVg267DEd8m1ZypXLBnvN7SFxYwV7ndqSYGiN9TTpu",
  "decimals": 10,
  "fromChain": 8,
  "toChain": "(rounded 100000)",
}
`;

exports[`'acala' to 'interlay' using bridgeSDK cross-chain 'INTR' > fee 1`] = `"(rounded 0.002)"`;

exports[`'acala' to 'interlay' using bridgeSDK cross-chain 'INTR' > initial 1`] = `
{
  "address": "5FA9nQDVg267DEd8m1ZypXLBnvN7SFxYwV7ndqSYGiN9TTpu",
  "decimals": 10,
  "fromChain": 10,
  "toChain": 100000,
}
`;
